specification/hybridconnectivity/HybridConnectivity.Management/client.tsp

```diff
@@ -90,7 +90,7 @@ namespace Microsoft.HybridConnectivity;
 
 @@clientName(Microsoft.HybridConnectivity, "HybridConnectivityMgmt", "python");
 @@clientName(Microsoft.HybridConnectivity,
-  "HybridConnectivityMgmtClient",
+  "HybridConnectivityManagementApi",
```

Comment:
No need to do this. `HybridConnectivityMgmtClient` is fine.

Reply:
Rollbacked to [2a61ad4](https://github.com/Azure/azure-rest-api-specs/pull/32243/commits/2a61ad4370ca8c35ca8cb3c8729c05b3df95a276)